### DISCUSSION TITLE
Miller 6.12.0

### DISCUSSION THREAD

**Discussion by johnkerl** [johnkerl/miller#1530] [Announcements]
## Features

* New `mlr sparsify` verb by @johnkerl in https://github.com/johnkerl/miller/pull/1498
* Support thousands separator in `fmtnum` by @johnkerl in https://github.com/johnkerl/miller/pull/1499
* Add descriptions for `put` and `filter` verbs by @johnkerl in https://github.com/johnkerl/miller/pull/1529

## Bugfixes

* Miller produces no output on TSV with > 64K characters per line by @johnkerl in https://github.com/johnkerl/miller/pull/1505
* Enable record-hashing by default by @johnkerl in https://github.com/johnkerl/miller/pull/1507
* Improved file-not-found handling by @johnkerl in https://github.com/johnkerl/miller/pull/1508
* Avoid spurious `[]` on JSON output in some cases by @johnkerl in https://github.com/johnkerl/miller/pull/1528

## Internal

* 6.11.0-dev by @johnkerl in https://github.com/johnkerl/miller/pull/1484
* Separate out `ILineReader` abstraction by @johnkerl in https://github.com/johnkerl/miller/pull/1504

## Dependency updates

* Bump actions/upload-artifact from 4.2.0 to 4.3.0 by @dependabot in https://github.com/johnkerl/miller/pull/1483
* Bump github.com/klauspost/compress from 1.16.7 to 1.17.5 by @dependabot in https://github.com/johnkerl/miller/pull/1486
* Bump actions/upload-artifact from 4.3.0 to 4.3.1 by @dependabot in https://github.com/johnkerl/miller/pull/1491
* Bump github.com/klauspost/compress from 1.17.5 to 1.17.6 by @dependabot in https://github.com/johnkerl/miller/pull/1492
* Bump golang.org/x/term from 0.16.0 to 0.17.0 by @dependabot in https://github.com/johnkerl/miller/pull/1494
* Bump github.com/klauspost/compress from 1.17.6 to 1.17.7 by @dependabot in https://github.com/johnkerl/miller/pull/1502
* Bump actions/cache from 4.0.0 to 4.0.1 by @dependabot in https://github.com/johnkerl/miller/pull/1511
* Bump github.com/stretchr/testify from 1.8.4 to 1.9.0 by @dependabot in https://github.com/johnkerl/miller/pull/1516
* Bump golang.org/x/sys from 0.17.0 to 0.18.0 by @dependabot in https://github.com/johnkerl/miller/pull/1521
* Bump golang.org/x/term from 0.17.0 to 0.18.0 by @dependabot in https://github.com/johnkerl/miller/pull/1522
* Bump actions/checkout from 4.1.1 to 4.1.2 by @dependabot in https://github.com/johnkerl/miller/pull/1526

**Full Changelog**: https://github.com/johnkerl/miller/compare/v6.11.0...v6.12.0

<hr /><em>This discussion was created from the release <a href='https://github.com/johnkerl/miller/releases/tag/v6.12.0'>Mil</a>.</em>

**Comment by aborruso** [comment]
Thank you very much
